jangjuseong/models/userDao.js

```diff
@@ -0,0 +1,40 @@
+const mysqlDataSource = require('./daoModule');
+
+const createUser = async (name, email, password, profileImage) => {
+  await mysqlDataSource.query(
+    `
+    INSERT INTO users (
+      name, 
+      email, 
+      password, 
+      profile_image
+    )
+    VALUES (
+      ?, 
+      ?, 
+      ?, 
+      ?
+    );
+    `,
+    [name, email, password, profileImage]
+  );
+};
+
+const signInUser = async (email) => {
```

Comment:
쿼리문을 통해 반환하는 값 자체는 유저에 대한 정보이기 때문에 아래와 같이 수정해주세요.

```suggestion
const getUser = async (email) => {
```